compiler/src/main/kotlin/io/spine/protodata/renderer/InsertionPoint.kt

```diff
@@ -26,13 +26,28 @@
 
 package io.spine.protodata.renderer
 
+import com.google.common.flogger.StackSize.FULL
+import com.google.protobuf.Empty
+import io.spine.logging.Logging.loggerFor
+import io.spine.protodata.FileCoordinates
```

Comment:
Why is it `FileCoordinates`, but not `TextCoordinates` or `SourceCodeCoordinates`?

A file could be binary, what if it's not a file at all? What if it's a piece of printed text, and we help Elon to sort out Twitter developers?.. :) Would these things apply to a paper roll?

Reply:
@dmdashenkov, `FileCoordinates` and `SpecCase` naming issues are better to resolve in this PR.

If we need to gather and discuss it, let's do it.